assets/js/src/core/components/version-accordion/version-accordion.tsx

```diff
@@ -68,19 +68,20 @@ export const VersionAccordion = ({
 
```

Comment:
I'm still struggling with this component. How do we reuse it with versions in the name?
And props tailored so specific for versions?

I think we need a new component for the versions. Something like TimelineAccordions.
API vise we could add props for items (List combined out of our new accordion component and the necessary timeline props), selectedKeys, selectable, etc.

Reply:
@vin0401 ok - I saw this more as just a refactoring of what was there and not as a new shared component to use. 
I thought this is to big and specific of a component to be made shareable? Likely I am just missing the full picture here and dont understand why that would be useful.

Reply:
In general we are not really limited in the size of the components. I would see that as a typical organism in regards to atomic design. 

Making something reusable in regards of UI is never a bad approach. Maybe we don't have a reusable case, yet. But maybe it will come up in one of our bundles or the community will find a practical use for it. 

And is it really that specific? Just imagine it without the content. It's just a timeline that takes accordions as a child.
How the content will look like in the end should be something that is up for the developer who uses the component. Content he could plug & play together out of other atoms/molecules/organism.  

Hope that helps? Otherwise you can call me anytime :P

Reply:
can we merge this one and I open a new pull request for this ? :)

Reply:
Yeah sure, just marked one last small thing. :) Can you create an issue for this topic? :)